components/ui/Timestamp.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {View} from 'react-native';
+import {Text} from '../ui/Text';
+
+export const Timestamp: React.FC<TimestampProps> = props => {
+  function formattedDate(): React.ReactNode {
+    const months = [
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+    ];
+
+    const date = new Date(props.time);
+    const day = date.getDate();
+    const month = months[date.getMonth()];
```

Comment:
use date.toLocaleString('default', { month: 'long' });

Reply:
this will support different locales too, we can use it later

Reply:
Done, updating with this